README.md

```diff
@@ -2,11 +2,13 @@
 
 ## Quick start
 
-1. Install the python package django-tailwind from pip
+1. Install the `django-tailwind` package via Pip:
 
-   `pip install django-tailwind`
+   `python -m pip install django-tailwind`
```

Comment:
Why did you change this? Isn't it more verbose?

Reply:
Apologies for not explaining. The short answer is because (1) [it eliminates confusion regarding which Python interpreter will be invoked](https://github.com/pypa/pip/issues/3164#issue-109993120), and (2) [Python itself switched to this recommended invocation](https://docs.python.org/3/installing/index.html#basic-usage).

Reply:
Thanks for the links, although I don't like it and never use it, I'd rather be in sync with Python docs.

Reply:
I don't use the `python -m pip` syntax either, but then again I only invoke Pip from within virtual environments, and thus it's always clear which interpreter I am using. Less-experienced folks are more likely to be bitten by bare `pip` invocations, which is why I believe this change in the docs is the right way to go. 👍